doc/api/timers.md

```diff
@@ -61,6 +61,21 @@ timer is active. Each of the `Timeout` objects returned by these functions
 export both `timeout.ref()` and `timeout.unref()` functions that can be used to
 control this default behavior.
 
+### timeout.refresh()
```

Comment:
The methods should be arranged alphabetically, so this entry should come after `timeout.ref()`, not before it.